src/main/java/mekanism/client/ClientTickHandler.java

```diff
@@ -123,6 +125,11 @@ public static boolean isVisionEnhancementOn(Player player) {
         return module != null && module.isEnabled() && module.getContainerEnergy().greaterThan(MekanismConfig.gear.mekaSuitEnergyUsageVisionEnhancement.get());
     }
 
+    public static boolean isHydroEnhancementOn(Player player) {
+        IModule<ModuleHydrostaticRepulsorUnit> module = MekanismAPI.getModuleHelper().load(player.getItemBySlot(EquipmentSlot.LEGS), MekanismModules.HYDROSTATIC_REPULSOR_UNIT);
+        return module != null && module.isEnabled() && module.getInstalledCount() == 4 && module.getCustomInstance().isSwimBoost() && module.getContainerEnergy().greaterThan(MekanismConfig.gear.mekaSuitEnergyUsageHydrostaticRepulsion.get());
```

Comment:
I feel like the check for the installed count could potentially be moved into the `isSwimBoost` method

Reply:
I didn't want to add the module param to isSwimBoost and it was having trouble finding the getInstalledCount method on itself so I just left it. If you figure out the right way to call it from its own instance I'm definitely willing to make that change though.

Reply:
Ahh you probably need to do `module.getCustomInstance().isSwimBoost(module)` similar to how most methods in `ICustomModule` take an `IModule` param

Reply:
Yup, just realized the Enchant Module class doesn't extend or implement Module in any way... That was confusing. You still think it's worth it though, even with adding the module param? It does clean up the stack size checks.

Reply:
Yeah probably